sync/http/component.go

```diff
@@ -121,3 +122,149 @@ func (c *Component) createHTTPServer() *http.Server {
 		Handler:      routerAfterMiddleware,
 	}
 }
+
+// propSetMsg is used to log successes when using
```

Comment:
Maybe use a more descriptive name in order to avoid the comment.

Reply:
Done. I opted to use `fieldSetMsg`, as we're setting the struct *fields*. I think it looks better.